dev/server

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env node
```

Comment:
Can we have something to run that automatically like `npx grunt dev` or similar and edit the `README.md` to mention it?

Reply:
It's available at `npm run dev` - I added this to the README.md as well.